canopeum_backend/canopeum_backend/serializers.py

```diff
@@ -580,6 +590,58 @@ def get_sponsors(self, obj) -> list[str]:
         return [batch.sponsor for batch in batches if batch.sponsor]
 
 
+class SiteDetailSummarySerializer(serializers.ModelSerializer[Site]):
```

Comment:
The API is called `SiteSummaryDetailAPIView`, should we call this 
```suggestion
class SiteSummaryDetailSerializer(serializers.ModelSerializer[Site]):
```